src/classes/HH_ManageHH_CTRL.cls

```diff
@@ -41,22 +41,57 @@ public with sharing class HH_ManageHH_CTRL {
 
     /** @description The Household Object or Account */
     public SObject hh { get; set; }
+    
+    /** @description The household cast to an Account for account override checking */
+    public Account hhA {
+        get {
+            return (Account)hh;
+        }
+    }
 
     /** @description what is the namespace (null if unmanaged, or npsp if managed) */
     public string namespacePrefix { get; set; }
     
     /*******************************************************************************************************
+    * @description to easily tell our Account override page whether to defer to the normal Account edit page
+    * when the user is trying to change their recordtype!
+    */
+    public boolean isChangingRecordType {
+        get {
+            if (ApexPages.currentPage() != null) {
+                map<string, string> mapParams = ApexPages.currentPage().getParameters();
+                if (mapParams != null) {
+                    string paramRtId = mapParams.get('RecordType');
+                    if (paramRtId != null && paramRtId != ACCT_IndividualAccounts_TDTM.rtIdForAccountModel(true)) {
+                        return true;
+                    }
+                }
+            }
+            return false; 
+        }
+    }
+    
+    /*******************************************************************************************************
     * @description Constructor for the page.  Loads up the HH Account or HH Object, using fields from
     * the appropriate field set.
     */
     public HH_ManageHH_CTRL() {
+        this(null);
+    }
+    
+    public HH_ManageHH_CTRL(ApexPages.StandardController controller) {
         namespacePrefix = UTIL_Namespace.getNamespace();
 
         Map<String, String> pageParams = ApexPages.currentPage().getParameters();
         hhId = pageParams.get('Id');
```

Comment:
Since you have `StandardController` passed in now, you could just do `controller.getId()`